### PR TITLE
[chore] fix migration test linting

### PR DESCRIPTION
**Pull request by benmccann**
lint is failing on https://github.com/sveltejs/kit/pull/5861. this should fix it

### CODE REVIEW

File: packages/migrate/migrations/routes/migrate_page_server/index.spec.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ import { migrate_page_server } from './index.js';
 
 for (const sample of read_samples(import.meta.url)) {
 	test(sample.description, () => {
+		if (!sample.filename) {
+			throw new Error('Missing filename for sample');
+		}
```
Comment (Rich-Harris) [inline packages/migrate/migrations/routes/migrate_page_server/index.spec.js:10]:
```suggestion
```

File: packages/migrate/migrations/routes/migrate_page_server/index.spec.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ import { migrate_page_server } from './index.js';
...
 		const actual = migrate_page_server(sample.before, sample.filename);
```
Comment (Rich-Harris) [inline packages/migrate/migrations/routes/migrate_page_server/index.spec.js:11]:
```suggestion
		const actual = migrate_page_server(sample.before, sample.filename ?? '+page.server.js');
```

File: packages/migrate/migrations/routes/migrate_page_server/index.spec.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { migrate_page_server } from './index.js';
 
 for (const sample of read_samples(import.meta.url)) {
 	test(sample.description, () => {
-		const actual = migrate_page_server(sample.before, sample.filename);
+		const actual = migrate_page_server(sample.before, sample.filename ?? '+page.server.js');
```
Comment (benmccann) [inline packages/migrate/migrations/routes/migrate_page_server/index.spec.js:8]: Should this defaulting be done inside the method? I.e. make the second parameter optional

Comment (Rich-Harris) [reply packages/migrate/migrations/routes/migrate_page_server/index.spec.js:8]: this code is saying that all samples implicitly _do_ have a filename, `+page.server.js`, unless one is specified. making the parameter optional for the sake of the tests is the wrong solution

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**